distributor-api-v1/operations.md

```diff
@@ -151,20 +151,107 @@ Preferred initial call used to obtain all static data about distributor configur
+| `ChildSelectionEnabled` | boolean | optional | Information whether enterprise wants to include an option to add children to reservations (children are accepted in reservations by default). |
+| `DisplayAvailability` | boolean | optional | Information whether enterprise wants to show your property's availability next to maximum occupancy in each space category (availability will be shown by default). |
+| `PaymentCardInput` | string [PaymentCardInput](operations.md#paymentcardinput) | required | Information how does enterprise want to handle payment cards. |
+| `RequiredFields` | array of [RequiredField](operations.md#requiredfield) | required | Fields that are set up as required in distributor configuration. |
+| `OnlineTravelAgencies` | array of string | required | List of travel agencies enterprise wants to include in comparison banner. |
+| `CompetitorRateDescription` | [LocalizedText](operations.md#localizedtext) | required | Features that enterprise set up which differentiate it's online booking from competitors (for example, `20% online booking discount` or `Breakfast included`). |
+| `CompetitorPriceRelativeAdjustment` | number | optional | Percentage that competitor's prices (listed in the rate comparison banner if `DisplayRateComparison` is set to `true`) will be shown as, compared to enterprise's Best Available Rate (BAR). For example, if enterprise's BAR costs 50, and entered here is "100", their rate will be shown as 50. If here is entered "110", their rate will be shown as 55 (as here is added a 10% markup). |
+| `Enterprise` | [Enterprise](operation.md#enterprise) | required | Enterprise that this `Configuration` belongs to. |
+
+#### PaymentCardInput
+
+* `NotRequested` - enterprise does not want to request payment card info.
+* `Requested` - enterprise wants a field for customers to enter their card details into, but does not want the card details to be validated.
+* `Required` - enterprise wants a field for customers to enter their card details into but the card must be validated before the reservation can be created. 
+
+#### RequiredField
+
+* `Telephone` - telephone field is required for both [Customer](operations.md#customer) and [Booker](operations.md#booker)
+
+#### Enterprise 
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `Id` | string | required | Unique identifier of the an enterprise. |
+| `Name` | [LocalizedText](operations.md#localizedtext) | required | Enterprise name. |
+| `Description` | [LocalizedText](operations.md#localizedtext) | required | Enterprise Description. |
+| `Address` | [Address](operation.md#address) | required | Address of the enterprise. |
+| `CityId` | string | required | Unique identifier of the city enterprise is in. |
+| `ImageId` | string | optional | Unique identifier of logo of the enterprise. |
```

Comment:
Logo? where is it defined as logo? Which logo?

Reply:
It's setup here:

![image](https://user-images.githubusercontent.com/6526378/78139280-89584a00-7428-11ea-870d-2d65fa51113c.png)

Is it ok or could you help with a better description?